worker/src/queues/ses/workers/sesSendHandler.ts

```diff
@@ -39,12 +40,31 @@ export async function sesHandler(job: Job<SESJob>) {
     },
   });
 
+  let response;
+
   try {
-    const response = await sesClient.send(emailCommand);
+    response = await sesClient.send(emailCommand);
     logger.info(`Reference ${reference} staff email sent successfully with SES message id: ${response.MessageId}`);
-    return response;
   } catch (err: SESServiceException | any) {
     logger.error({ jobId, reference, err }, "SES could not send the email");
     throw err;
   }
+
+  sendAlertToPost(job);
+  return response;
+}
+
+/**
+ * Sends an alert to individual posts, notifying them that a form has been submitted to the shared inbox.
+ */
+export async function sendAlertToPost(job: Job<SESJob>) {
```

Comment:
Same with this, I'm guessing we might need to rethink how we do our separation if we wanted to have all staff emails being handled in the same place and all user emails handled in another place if some are going through notify